### PR TITLE
Initial restructuring.

### PR DESCRIPTION
**Pull request by samcunliffe**
:pear: dev with @sfmig #TeamWork

Bring over the structure from [neuroinformatics-unit/python-cookiecutter](https://github.com/neuroinformatics-unit/python-cookiecutter). Make the code into an importable python package. For now the code is dumped into the `crabs` module. So everything needs to be of the form `from crabs.bboxes_labelling import ...`.

## Solves:

* #62 
* #63 

## Begins to fix:

* #73

### CODE REVIEW

File: crabs/dense optical flow/estimate_optical_flow_on_video.py
Comment (samcunliffe) [inline crabs/dense optical flow/estimate_optical_flow_on_video.py:1]: Shall we rename `dense optical flow` → `dense_optical_flow` already here?
Feels like it'd save some pain for future us.

File: crabs/pose estimation/cluster bash scripts/howto_inference_cluster.md
Comment (samcunliffe) [inline crabs/pose estimation/cluster bash scripts/howto_inference_cluster.md:1]: Ditto here, probably `pose estimation` → `pose_estimation`.

**Review by samcunliffe — COMMENTED** [review]
I can't approve, but I don't see any bugs so far.
I'm good to merge if you are.

File: pyproject.toml [hunk 1]
```diff
@@ -1,126 +1,127 @@
...
+
 classifiers = [
-    "Operating System :: POSIX",
+    "Development Status :: 2 - Pre-Alpha",
+    "Programming Language :: Python",
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3.9",
-    "Typing :: Typed",
+    "Programming Language :: Python :: 3.10",
+    "Operating System :: OS Independent",
```
Comment (sfmig) [inline pyproject.toml:21]:
```suggestion
    "Operating System :: POSIX",
```
since we don't test Windows?

**Review by sfmig — APPROVED** [review]
thanks @samcunliffe! Just a small comment about the project metadata in pyproject.toml and the OS support.

If you give me the ok I'll change it and merge

File: pyproject.toml (hunk 1 above)
Comment (samcunliffe) [reply pyproject.toml:21]: Yes, and we're developing stuff 100% targetted for the SWC cluster.

**Pull request merged**